pom.xml

```diff
@@ -48,6 +60,22 @@
         </repository>
     </distributionManagement>
 
+    <repositories>
```

Comment:
Maybe add a comment here just to note that once Spring is final we can remove this.

Reply:
yes sir!